app/client/src/components/ProfileInfo.jsx

```diff
@@ -28,8 +28,12 @@ import { Typography, Box, Stack, IconButton, Divider } from "@mui/material";
 import EditRoundedIcon from "@mui/icons-material/EditRounded";
 import { openEditUserInfoModal } from "../redux/ducks/modalDuck";
 import EditUserInfoModal from "./modals/EditUserInfoModal";
+import { useTheme } from "@emotion/react";
 
 const ProfileInfo = (props) => {
+  const theme = useTheme();
+  const neutralTextColor = theme.palette.neutral.main;
```

Comment:
Can redefine this to: "neutral.main"

The 'sx' property is theme-aware as long as the appropriate string value is used. This means we can remove useTheme().